src/randoop/operation/DummyStatement.java

```diff
@@ -6,11 +6,16 @@
 import java.util.List;
 
 import randoop.ExecutionOutcome;
-import randoop.Globals;
 import randoop.NormalExecution;
 import randoop.sequence.Variable;
 
-public class DummyStatement implements Operation, Serializable {
+/**
+ * DummyStatement is an {@link Operation} representing a noop.
+ * As an operation a DummyStatement with name noop is represented as 
+ * <i>noop</i> : [] &rarr; void, 
+ * an operation with no input types and void output. It does nothing when executed.
+ */
```

Comment:
Can you remark on why is is useful or how it is used?
(This comment is already a big improvement over the lack of comments in the previous version of the code, of course.)